source/Generic/PlayState/Models/PlayStateData.cs

```diff
@@ -58,6 +63,14 @@ private void Settings_PropertyChanged(object sender, System.ComponentModel.Prope
             {
                 SetSuspendMode();
             }
+            if (e.PropertyName == nameof(settingsModel.Settings.NotificationPlayingReminderMinutes))
+            {
+                PlayingReminderTimer.Interval = TimeSpan.FromMinutes(settingsModel.Settings.NotificationPlayingReminderMinutes);
+            }
+            if (e.PropertyName == nameof(settingsModel.Settings.NotificationSuspensionReminderMinutes))
+            {
+                SuspensionReminderTimer.Interval = TimeSpan.FromMinutes(settingsModel.Settings.NotificationSuspensionReminderMinutes);
+            }
```

Comment:
I don't like this design, I think the object itself shouldn't take care of what happens outside of it. The `PlayStateData` object should only be in charge of holding data and not have functionality. In my opinion the `PlayState Manager` should be in charge of handling the notification events. If possible please design this another way